### ISSUE TITLE
Drag / Scroll prompt when Hovering

### ISSUE THREAD

**Issue by ramsesgtz42** [aradzie/keybr.com#225]
**Add Hover Text box when hovering Keyboard UI**

When the Keyboard / Text Field UI is hovered over, there is no indication it can be dragged or scrolled to be moved/resized. I think a text box when hovered would be useful, just like all the rest of the hover text boxes on the UI. I didn't know it was possible to resize the UI until I accidentally scrolled on it, and I think it would help people with visibility issues.

*Attached image of example text box, in case wasn't clear*

![firefox_NogTV3LkMX](https://github.com/aradzie/keybr.com/assets/101148268/e7a82018-cdfc-49e4-89e3-8017ce04014f)

**Comment by aradzie** [comment]
Yes, It's not easy to discover this feature. My best effort so far is to flash a box with a resize handle around and element on hover. But it needs to be improved.
By the way, Ctrl+Click on an element restores its original position/zoom.
